README.md

```diff
@@ -74,6 +74,56 @@ EOF
 kubectl create -f <rdma-app.yaml>
 ```
 
+# RDMA Shared Device Plugin Configurations
+The plugin has several configuration fields, this section explains each field usage
+
+```json
+{
+  "configList": [{
+    "resourceName": "hca_shared_devices_a",
+    "rdmaHcaMax": 1000,
+    "devices": ["ib0", "ib1"]
+  },
+    {
+      "resourceName": "hca_shared_devices_b",
+      "rdmaHcaMax": 500,
+      "selectors": {
+        "vendors": ["15b3"],
+        "deviceIDs": ["1017"],
+        "ifNames": ["ib3", "ib4"]
+      }
+    }
+  ]
+}
+```
+
+`"resourceList"` should contain a list of config objects. Each config object may consist of following fields:
+
+
+|     Field      | Required |                                                    Description                                                    |     Type      |                         Example                         |
+|----------------|----------|-------------------------------------------------------------------------------------------------------------------|---------------|---------------------------------------------------------|
+| "resourceName" | Y        | Endpoint resource name. Should not contain special characters, must be unique in the scope of the resource prefix | string        | "hca_shared_devices_a"                                  |
+| "rdmaHcaMax"   | Y        | Maximum number of RDMA resources that can be provided by the device plugin                                        | Integer       | 1000                                                    |
+| "selectors"    | N        | A map of device selectors for filtering the devices                                                               | json object   | selectors": {"vendors": ["15b3"],"deviceIDs": ["1017"]} |
```

Comment:
> A map of device selectors for filtering the devices

A map of device selectors for filtering the devices. refer to (Device Selectors)[add link] section for more information